README.md

```diff
@@ -290,6 +302,19 @@ Manual DNS configuration can be specified via a list of addresses
  properties, respectively. If specified, it determines the route metric
 for DHCP assigned routes and the default route, and thus the priority for multiple interfaces.
 
+Static route configuration can be specified via a list of routes given in the `route`
+option. The default value is an empty list. Each route is a dictionary with the following
+entries: `network`, `prefix`, `gateway` and `metric`. `network` and `prefix` together specify
+the destination network. CIDR notation or network mask notation are not supported yet. If the
+boolean option `route_append_only` is `True`, the specified routes are appended to the
+existing routes, if it is `False` (default), the current routes are replaced. Setting this
+option to `True` without setting `route` has the effect of preserving the current static routes. The
```

Comment:
and
    option to `no` without setting `route` has the effect of preserving the current static routes. The

Reply:
I believe you meant
option to `yes` without setting route has the effect of preserving the current static routes. 
`no` makes no sense here IMHO.

Reply:
yes, good catch!